src/newrelic_telemetry_sdk/client.py

```diff
@@ -94,7 +94,7 @@ class Client(object):
     Usage::
 
         >>> import os
-        >>> insert_key = os.environ.get("NEW_RELIC_INSERT_KEY", "")
```

Comment:
To minimize confusion, maybe we can update the variable name to be `license_key` also.

Reply:
Whoops, that is very valid.  Changing it now.